source/extensions/filters/http/transformation/inja_transformer.cc

```diff
@@ -390,7 +391,16 @@ json TransformerInstance::base64_encode_callback(const inja::Arguments &args) co
 
 json TransformerInstance::base64_decode_callback(const inja::Arguments &args) const {
```

Comment:
Should we also introduce a callback that is strict base64?
Also can we add a comment with a link to the rfc? https://datatracker.ietf.org/doc/html/rfc4648#section-5

Reply:
good call added.

I was originally trying to surgically fix the problem, but the add wasn't a big lift.

A few tests would be useful but I really don't have time right now.

Reply:
I would prefer to keep current impl as is to avoid changing behavior..m

Reply:
I have reverted the change to the current impl so we only have net-new. The test now tests the `base64url_decode` callback also